scene/gui/texture_progress_bar.cpp

```diff
@@ -669,6 +671,16 @@ Point2 TextureProgressBar::get_radial_center_offset() {
 	return rad_center_off;
 }
 
+void TextureProgressBar::_validate_property(PropertyInfo &p_property) const {
+	if (p_property.name.begins_with("stretch_margin") && !nine_patch_stretch) {
```

Comment:
```suggestion
	if (p_property.name.begins_with("stretch_margin_") && !nine_patch_stretch) {
```
I believe we normally add the `_` to ensure its a prefix